MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm

```diff
@@ -2328,6 +2328,11 @@
 	}
 #endif
 
+#if MVK_XCODE_15
+	if (mslVersionIsAtLeast(MTLLanguageVersion3_1)) {
+		_metalFeatures.nativeTextureAtomics = true;
+	}
+#endif
```

Comment:
Combine this section with existing `MVK_XCODE_15`:

https://github.com/KhronosGroup/MoltenVK/blob/37361ccd5595ff08c050981856903713f96221d2/MoltenVK/MoltenVK/GPUObjects/MVKDevice.mm#L1810C1-L1814C1